Base/home/anon/tests/run-tests-and-shutdown.sh

```diff
@@ -6,26 +6,49 @@ echo "==== Running Tests on SerenityOS ===="
 run(index) {
     shift
     test_cmd=($*)
-    echo "Running test $index out of $count_of_all_tests -- $test_cmd"
+    echo "==== Running test $index out of $count_of_all_tests -- $test_cmd ===="
+    echo "::group::$test_cmd"
     if $test_cmd {
-        echo "::debug file=$test_cmd:: $test_cmd passed!"
+        echo "::endgroup::"
+        echo "==== $test_cmd passed! ===="
     } else {
+        echo "::endgroup::"
+        failed_tests=${concat_lists $failed_tests ($test_cmd)}
+        echo "==== Added $test_cmd to list of failed tests. Failed tests so far: $failed_tests ===="
         echo "::error file=$test_cmd:: $test_cmd returned non-zero exit code, check logs!"
     }
 }
 
-# TODO: test-web requires the window server
-system_tests=((test-js --show-progress=false) test-pthread /usr/Tests/LibM/test-math (test-crypto -t bigint))
-# FIXME: Running too much at once is likely to run into #5541. Remove commented out find below when stable
-all_tests=${concat_lists $system_tests} #$(find /usr/Tests -type f | grep -v Kernel | grep -v .inc | shuf))
+# Files in /usr/Tests/* that we don't want to execute match these patterns:
+#     Kernel: Kernel tests that are either old patched exploits, or hang the system/shell
```

Comment:
I don't think this is true anymore... there are tests under `/usr/Tests/Kernel/` that start with Test* which use the LibTest framework and should be passing.